ads/mediavine.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function mediavine(global, data) {
+  validateData(data, ['site'], ['sizes']);
```

Comment:
@ehoch FYI: 
I don't know how you read the data.sizes. Because I don't see how you pass `data.sizes` when you call `loadScript`
But we have a recent change that we move data from url fragment to name attribute.  If you are reading the `data.sizes` from url directly, it will no longer be available. 
Your example ad and it looked fine. But please double check before the change goes to PROD. The change is in canary today, and will be live in a week.